.github/workflows/nightly_tests.yml

```diff
@@ -0,0 +1,99 @@
+name: Nightly Tests
+
+on: [push]
+#  schedule:
+#    cron: '* 8 * * * *'
```

Comment:
Note, `push` was needed to debug here. I'll re-enable the cron before we merge this.